site/js/deregistration.js

```diff
@@ -3,33 +3,31 @@ import { _ } from './modules/gettext'
 import { gettext } from './modules/gettext'
 import PageScript from './modules/script.js'
 import { setup_the_deregister_form_buttons } from './modules/setup_buttons'
-
+import * as Ajax from './modules/ajax'
```

Comment:
Igy minden fv elérhető a modulból adott esetben az Ajax.fv néven. És csak azok a fv-k rollupolódnak a deploy kódba, amiket használunk is, nem az egész hóbelebanc